runtime/jvmti/jvmtiThread.c

```diff
@@ -665,28 +663,50 @@ jvmtiGetOwnedMonitorInfo(jvmtiEnv* env,
 		ENSURE_NON_NULL(owned_monitors_ptr);
 
 		rc = getVMThread(currentThread, thread, &targetThread, TRUE, TRUE);
-		if (rc == JVMTI_ERROR_NONE) {
-			jobject * locks;
+		if (JVMTI_ERROR_NONE == rc) {
+			jobject *locks = NULL;
 			jint count = 0;
+			J9VMThread *threadToWalk = targetThread;
 
-			vm->internalVMFunctions->haltThreadForInspection(currentThread, targetThread);
+#if JAVA_SPEC_VERSION >= 19
+			J9VMThread stackThread = {0};
+			J9VMEntryLocalStorage els = {0};
```

Comment:
This will not work as `ownedMonitorIterator` which gets invoked in `walkLocalMonitorRefs` checks the owner thread address to determine if the walkThread is the owner. see [line 1233](https://github.com/eclipse-openj9/openj9/blob/master/runtime/jvmti/jvmtiThread.c#L1163-L1164)